### ISSUE TITLE
Obtain the VMAF (Video Multi-Method Assessment Fusion) score between two input videos.

### ISSUE THREAD

**Issue by brycechesternewman** [jrottenberg/ffmpeg#257]
Hello,
Is there a reason why (perhaps legal or just not considered) that https://ffmpeg.org/ffmpeg-filters.html#libvmaf can not be added?
Thank you.

**Comment by brycechesternewman** [comment]
I can give it a try. This is the file you are talking about https://github.com/jrottenberg/ffmpeg/pull/127/files? Which will need a VMAF version upgrade of course.

**Comment by jrottenberg** [comment]
Right basically https://github.com/jrottenberg/ffmpeg/pull/127/commits/89295bb32d2b8701539286c05f25be52b9bdc616 was correct,

1. update the templates accordingly (Dockerfile-env and Dockerfile-run)
2. run ./update.py to propagate the changes on all variants/versions
3. test

**Comment by brycechesternewman** [comment]
Yep. Got some time to read through the code. Nice work on the Dockerfile-env and Dockerfile-run.

**Comment by brycechesternewman** [comment]
Do you need to give me permissions to push a branch? "remote: Permission to jrottenberg/ffmpeg.git denied to..."

**Comment by jrottenberg** [comment]
you need to create a fork, and push to your local repo, once you are happy with your changes, open Pull requests your_repo:your_branch against jrottenberg/ffmpeg:master
